### DISCUSSION TITLE
6.x 微信公众号网页授权授权码验证无任何报错

### DISCUSSION THREAD

**Discussion by sarices** [w7corp/easywechat#2774] [Q&A]
`
$code = "微信回调URL携带的 code"; // $_GET['code']
$user = $oauth->userFromCode($code);
`
这一步我把配置都写错了，但实际上执行的时候只是返回了一个对象，里面的所有key 都是 null 的，希望能抛出个异常，好让程序判断是什么原因导致了问题

**Comment by iluyuns** [comment]
->getRaw();  推测你是静默授权 出的错误，静默授权 需要你  $oauth->scopes(['snsapi_base'])  在调用 userFromCode($code);
